source/index.js

```diff
@@ -234,7 +260,13 @@ module.exports = async (input = 'patch', options) => {
 	tasks.add({
 		title: 'Creating release draft on GitHub',
 		enabled: () => isOnGitHub === true,
-		skip: () => !options.releaseDraft,
+		skip: () => {
+			if (options.preview) {
+				return 'GitHub Releases draft not opened in preview mode.';
```

Comment:
Shouldn't this have the `[Preview]` prefix?

Reply:
Maybe 'GitHub Releases draft **will** not **be** opened in preview mode.'.